modules/platforms/python/pyignite/cache.py

```diff
@@ -185,10 +205,116 @@ def destroy(self):
         """
         Destroys cache with a given name.
         """
-        return cache_destroy(self._client, self._cache_id)
+        return cache_destroy(self.get_best_node(), self._cache_id)
 
     @status_to_exception(CacheError)
-    def get(self, key, key_hint: object=None) -> Any:
+    def _get_affinity(self, conn: 'Connection') -> Dict:
+        """
+        Queries server for affinity mappings. Retries in case
+        of an intermittent error (most probably “Getting affinity for topology
+        version earlier than affinity is calculated”).
+
+        :param conn: connection to Igneite server,
+        :return: OP_CACHE_PARTITIONS operation result value.
+        """
+        for _ in range(AFFINITY_RETRIES or 1):
+            result = cache_get_node_partitions(conn, self._cache_id)
+            if result.status == 0 and result.value['partition_mapping']:
+                break
+            time.sleep(AFFINITY_DELAY)
+
+        return result
+
+    @select_version
+    def get_best_node(
```

Comment:
Is it public API? If yes, then why do we provide a user API to get node?

Reply:
I added a note in the method's docstring to discourage user from using this function.